changelog.d/9033.misc

```diff
@@ -0,0 +1 @@
+Allow bumping schema version when using split out state database.
```

Comment:
I put this as misc, rather than bugfix, as this doesn't effect anyone other than synapse devs.